packages/plugins/operators/operators-js/src/operators/shared/number.test.js

```diff
@@ -56,7 +56,7 @@ test('_number valid functions', () => {
   );
   expect(
     _number({ methodName: 'toLocaleString', params: [123456.789, 'de-DE'], location: 'locationId' })
-  ).toBe('123.456,789');
+  ).toBe('123,456.789');
```

Comment:
Why has this changed?

Reply:
Not sure. The method returned the new value so I updated it. I'm not sure if the old value passed in the previous tests?